YoutubeDownloader.Core/Downloading/FFmpeg.cs

```diff
@@ -17,7 +17,10 @@ static IEnumerable<string> GetProbeDirectoryPaths()
             yield return AppContext.BaseDirectory;
             yield return Directory.GetCurrentDirectory();
 
-            if (Environment.GetEnvironmentVariable("PATH")?.Split(Path.PathSeparator) is { } paths)
+            var UserAndMachineEnvironmentVariables =
+                $"{Environment.GetEnvironmentVariable("PATH", EnvironmentVariableTarget.User)}{Environment.GetEnvironmentVariable("PATH", EnvironmentVariableTarget.Machine)}";
+
+            if (UserAndMachineEnvironmentVariables?.Split(Path.PathSeparator) is { } paths)
```

Comment:
Please just do two `if`'s/`foreach`'s, it's easier to read this way

Reply:
> Please just do two `if`'s/`foreach`'s, it's easier to read this way

I've done with two if's/foreach's as your demand.
Please check.